VTUL/dlp-access
# update package version and readme

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2295) (:star:)

# What does this Pull Request do? (:star:)
1. Update package version to solve the security warnings
2. Update readme about deploying Amplify app using AWS CLI
3. remove README_react.md and  yarn.lock files

# How should this be tested?
1. Pull the code, start the local server, runt test suite and check if anything is working. 
2. You can see in Ampliy preview-dlp-access and security branch is deployed successfully.

# Interested parties
Tag (@VTUL/dld-dev ) interested parties

(:star:) Required fields
